### DISCUSSION TITLE
[Detector Support]: what Coral model suits best?

### DISCUSSION THREAD

**Discussion by Cr4z33** [blakeblackshear/frigate#13237] [Detector Support] [answered]
### Describe the problem you are having

Right now I am running Frigate as a Home Assistant integration in a VM in my Synology NAS for my Dahua video doorbell therefore I am not using the detection yet.

However in the next couple of months I am going to install 6 cameras outside and a Google Coral to speed up the detections.

Considering the amount of cameras would a Google Coral M.2 Accelerator B+M key connected to a USB 3.0 adapter be fair enough?

### Version

0.14.0-da913d8

### Frigate config file

```yaml
123
```

### docker-compose file or Docker CLI command

```yaml
123
```

### Relevant log output

```shell
123
```

### Operating system

HassOS

### Install method

HassOS Addon

### Object Detector

Coral

**Comment by hawkeye217** [comment, marked as answer]
Yes, a single Coral should handle that without issue.

**Comment by Cr4z33** [comment]
Thanks that was fast! 😁
